core/session.rb

```diff
@@ -91,6 +91,7 @@ class Session
   attr_accessor :all
   attr_accessor :isForce
   attr_accessor :force_version
+  attr_accessor :architecture
```

Comment:
Why did you add this argument to the MDBCI? Please describe the scenario that requires the use of the argument.